zabbix-agent.sls

```diff
@@ -0,0 +1,31 @@
+zabbix-agent:
+  '4.2.3.2400':
+    {% if grains['cpuarch'] == 'AMD64' %}
```

Comment:
The IF loop should be removed if you don't have a 32Bit installer or higher-level that if the system is not a 64Bit system, the entire state is bypassed